### PR TITLE
Update Demos to use Chime Meetings regional endpoints

### PR DESCRIPTION
**Pull request by harshitOberoi**
**Issue #:**
Currently JS SDK Demo just use Chime Meetings Global Endpoint. So using the current setup cannot create sauce lab test to test chime meetings regional endpoints.

**Description of changes:**
Update JS SDK Browser and Serverless Demo to include ChimeMeetingsWrapper using which we can swtich between global and regional endpoints using controlRegion url parameter.

**Testing:**

*Can these tested using a demo application? Please provide reproducible step-by-step instructions.*
yes.

We can deploy the serverless demo using below commands:
npm run deploy -- -r us-east-1 -b chime-meetings-sdk-demo-app-gamma -s chime-meetings-sdk-demo-app-prod-gamma -a meeting  -c https://tapioca.us-east-1.amazonaws.com -u false
npm run deploy -- -r us-east-1 -b chime-meetings-sdk-demo-app-gamma-r -s chime-meetings-sdk-demo-app-prod-gamma-r -a meeting -c meetings-gchime.us-east-1.amazonaws.com
npm run deploy -- -r eu-central-1 -b chime-meetings-sdk-demo-app-gamma-fra -s chime-meetings-sdk-demo-app-prod-gamma-fra -a meeting -c meetings-gchime.eu-central-1.amazonaws.com
npm run deploy -- -r ap-southeast-1 -b chime-meetings-sdk-demo-app-gamma-sin -s chime-meetings-sdk-demo-app-prod-gamma-sin -a meeting -c meetings-gchime.ap-southeast-1.amazonaws.com

**Checklist:**

1. Have you successfully run `npm run build:release` locally?
yes

2. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved?
no

3. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -19,6 +19,13 @@ jobs:
         AWS_DEFAULT_REGION: us-east-1
         AWS_DEFAULT_OUTPUT: text
         GAMMA_CHIME_ENDPOINT: ${{secrets.GAMMA_CHIME_ENDPOINT}}
+        GAMMA_CHIME_ENDPOINT_IAD: ${{secrets.GAMMA_CHIME_ENDPOINT_IAD}}
```
Comment (ltrung) [inline .github/workflows/deploy.yml:22]: Have you set this yet?

File: demos/browser/server.js [hunk 2]
```diff
@@ -19,23 +19,28 @@ console.info('Using index path', indexPagePath);
 
 const indexPage = fs.readFileSync(indexPagePath);
 
+
+// Set the AWS SDK Chime endpoint. The Chime endpoint is https://service.chime.aws.amazon.com.
+const endpoint = process.env.ENDPOINT || 'https://service.chime.aws.amazon.com';
+const currentRegion = process.env.Region || 'us-east-1';
```
Comment (ltrung) [inline demos/browser/server.js:25]: Need to update documentation on how to run local demo with region option

File: script/deploy-canary-demo [hunk 3]
```diff
@@ -9,7 +9,11 @@ cd $GITHUB_WORKSPACE/demos/serverless
 npm ci
 
 echo "Deploying to alpha stage for canary"
-npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -i eu-south-1 -t -l
+npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l
```
Comment (ltrung) [inline script/deploy-canary-demo:12]: This is only for standup and not canary so i think you can skip this.

File: script/deploy-canary-demo [hunk 4]
```diff
@@ -9,7 +9,11 @@ cd $GITHUB_WORKSPACE/demos/serverless
...
+npm run deploy -- -r us-east-1 -b chime-sdk-demo-canary-iad -s chime-sdk-demo-canary-iad -o chime-sdk-demo-canary-iad -c $CHIME_ENDPOINT_IAD -t -l
```
Comment (ltrung) [inline script/deploy-canary-demo:13]: Why do you need to pass in endpoint for this? It should be enough to just pass in region right?

**Review by ltrung — CHANGES_REQUESTED** [review]
Need update these scripts too:
- https://github.com/aws/amazon-chime-sdk-js/blob/master/script/release#L90 (QA test link).
- https://github.com/aws/amazon-chime-sdk-js/blob/master/script/postrelease#L41 (Prod canary)

File: demos/serverless/template.yaml [hunk 5]
```diff
@@ -7,6 +7,15 @@ Parameters:
     Default: false
     Type: String
     AllowedValues: [true, false]
+  Region:
+    Description: The AWS SDK Chime REGION
+    Default: "us-east-1"
+    Type: String
+  UseRegional:
+    Description: The AWS SDK Chime REGION
```
Comment (devalevenkatesh) [inline demos/serverless/template.yaml:15]: Need to update the description here.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (devalevenkatesh) [reply .github/workflows/deploy.yml:22]: We would need a better name and lets not use the region short names here.

File: script/deploy-canary-demo [hunk 6]
```diff
@@ -21,5 +25,8 @@ npm run deploy -- -b chime-sdk-meeting-readiness-checker-gamma-canary -s chime-s
 echo "Deploying to beta stage for canary that talks to gamma Chime endpoint"
 export AWS_ACCESS_KEY_ID=$BETA_AWS_ACCESS_KEY_ID
 export AWS_SECRET_ACCESS_KEY=$BETA_AWS_SECRET_ACCESS_KEY
-npm run deploy -- -b chime-sdk-demo-beta-canary -s chime-sdk-demo-beta-canary -o chime-sdk-demo-beta-canary -i eu-south-1 -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -t -l
+npm run deploy -- -b chime-sdk-demo-beta-canary -s chime-sdk-demo-beta-canary -o chime-sdk-demo-beta-canary -i eu-south-1 -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT -u false -t -l
+npm run deploy -- -r us-east-1 -b chime-sdk-demo-beta-canary-iad -s chime-sdk-demo-beta-canary-iad -o chime-sdk-demo-beta-canary-iad -p $GAMMA_CHIME_SERVICE_PRINCIPAL -c $GAMMA_CHIME_ENDPOINT_IAD -t -l
```
Comment (devalevenkatesh) [inline script/deploy-canary-demo:29]: Same applies for not using the region names like IAD, FRA outside like this so lets remove it.

Comment (devalevenkatesh) [inline script/deploy-canary-demo:29]: Can you please remove the short names and use regions in the naming?

File: script/deploy-canary-demo (hunk 3 above)
Comment (harshitOberoi) [reply script/deploy-canary-demo:12]: but if i dont update it the prod the standup link will use Chime Meetings SDK

**Pull request closed without merging**